src/components/player/player.tsx

```diff
@@ -432,6 +410,12 @@ export class Player {
   public async seek(seconds: number) {
     await this._invokePlayerFunction('seek', [seconds]);
 
+    this.status = {
+      ...this.status,
+      progress: { ...this.status.progress, seconds },
```

Comment:
So now we are only using seconds instead of seconds and percent, right?

I got to admit I don't understand what's really going on here :)

Reply:
Good catch! I don't think we ever access `percent`. Maybe we can simplify the entire `Progress` interface.

Reply:
yes, that's correct. This is using the [spread operator ](https://javascript.plainenglish.io/how-to-easily-update-your-objects-with-the-spread-operator-8f3dfdbcf956) to only update the `seconds` leaving that rest as it is.

The `percent` is already being updated in the [timeUpdate](https://github.com/openHPI/xikolo-video-player/blob/dev/src/components/player/player.tsx#L241) method, which will run almost* every time this method does 

*there is only one case where it does not and that's why we explicitly need to send the `seconds` here. The `percent` is not used in this case, so no need to pass it.

Reply:
> Maybe we can simplify the entire Progress interface by removing it.

yes, I also did not find any usage of it. Could be a follow-up.

Reply:
Good idea, let's refactor it as a separate commit on `dev` to keep the PR clean.

Reply:
The `percent` may have been used to indicate the watch progress in the course progress at some point. But for this, the percentage must be emitted via events or so. Good choice to drop it for now.